File: ultron/src/main/java/com/github/udoo/ultron/dao/domain/AccountDO.java
package com.github.udoo.ultron.dao.domain;

import com.baomidou.mybatisplus.annotations.TableField;
import com.baomidou.mybatisplus.annotations.TableId;
import com.baomidou.mybatisplus.annotations.TableName;
import com.baomidou.mybatisplus.enums.FieldFill;
import com.baomidou.mybatisplus.enums.FieldStrategy;
import com.baomidou.mybatisplus.enums.IdType;
import lombok.Data;

import java.util.Date;

/**
 * @author dong.yang
 * @data 2019/7/22 14:09
 */
@Data
@TableName("tm_account")
public class AccountDO {

    @TableId(value = "id", type = IdType.AUTO)
    private Long id;

    @TableField(fill = FieldFill.INSERT)
    private String email;

    @TableField
    private String name;

    @TableField(strategy = FieldStrategy.IGNORED)
    private String passwd;

    @TableField(fill = FieldFill.INSERT)
    private Integer creater;

    @TableField(value = "created_at", fill = FieldFill.INSERT)
    private Date createdAt;

    private Integer modifier;

    @TableField("updated_at")
    private Date updatedAt;
}

File: ultron/ultron-server/src/main/java/com/github/udoo/ultron/dao/domain/AccountDO.kt
package com.github.udoo.ultron.dao.domain

import com.baomidou.mybatisplus.annotations.TableField
import com.baomidou.mybatisplus.annotations.TableId
import com.baomidou.mybatisplus.annotations.TableName
import com.baomidou.mybatisplus.enums.FieldFill
import com.baomidou.mybatisplus.enums.FieldStrategy
import com.baomidou.mybatisplus.enums.IdType
import java.util.*

/**
 * @author dong.yang
 * @data 2019/7/23 9:53
 */
@TableName("tm_account")
class AccountDO {

    @TableId(value = "id", type = IdType.AUTO)
    var id: Long? = null

    @TableField(fill = FieldFill.INSERT)
    var email: String? = null

    @TableField
    var name: String? = null

    @TableField(strategy = FieldStrategy.IGNORED)
    var passwd: String? = null

    @TableField(fill = FieldFill.INSERT)
    var creater: Int? = null

    @TableField(value = "created_at", fill = FieldFill.INSERT)
    var createdAt: Date? = null

    var modifier: Int? = null

    @TableField("updated_at")
    var updatedAt: Date? = null
}